VirtoCommerce.LiquidThemeEngine/Filters/FeatureFilter.cs

```diff
@@ -0,0 +1,65 @@
+namespace VirtoCommerce.LiquidThemeEngine.Filters
+{
+    using System;
+
+    using global::Scriban;
+
+    using Newtonsoft.Json.Linq;
+
+    public class FeatureFilter
+    {
+        public static string IsFeatureActive(TemplateContext context, string key, params object[] variables)
+        {
+            if (string.IsNullOrEmpty(key))
+            {
+                return IsFeaturesActive(context, key, variables);
+            }
+
+            string result;
```

Comment:
this condition may be simplified by default initialization
```C#
var result = Convert.ToString(false);
if (context.TemplateLoader is ShopifyLiquidThemeEngine themeEngine)
{
...
}
``

Reply:
I simplified the entire code